pom.xml

```diff
@@ -37,7 +37,7 @@
     <okio.version>1.16.0</okio.version>
     <okio2.version>2.1.0</okio2.version>
     <kotlin.version>1.3.60</kotlin.version>
-    <kotlinpoet.version>1.4.4</kotlinpoet.version>
+    <kotlinpoet.version>1.5.0</kotlinpoet.version>
```

Comment:
I would imagine this brings in a lot of other breaking changes in metadata with the file facade support